### PR TITLE
Refactor InjCtrl IOC and InjSys devices

### PR DESCRIPTION
**Pull request by fernandohds564**
Move some tasks related to putting injection subsystems in standby or controlling warmup to the InjSysStandbyHandler class.

### CODE REVIEW

File: siriuspy/siriuspy/devices/injsys.py [hunk 1]
```diff
@@ -426,30 +431,58 @@ def cmd_turn_on(self):
...
+
+        if retval[0]:
+            return True, '', []
+        text = 'Check for BO Mags Triggers to be disabled timed '\
+               'out without success! Verify BO Mags Triggers!'
+        return [False, text, retval[1]]
+
+    def enable_triggers(self):
+        """."""
+        # disable triggers
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:459]: typo: enable

File: siriuspy/siriuspy/devices/injsys.py [hunk 2]
```diff
@@ -426,30 +431,58 @@ def cmd_turn_on(self):
...
             timeout=3, return_prob=True)
-        if not retval[0]:
-            text = 'Check for BO Mags Triggers to be enable timed '\
-                   'out without success! Verify BO Mags Triggers!'
-            return [False, text, retval[1]]
 
-        return True, '', []
+        if retval[0]:
+            return True, '', []
+        text = 'Check for BO Mags Triggers to be enable timed '\
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:470]: typo: to be enabled

File: siriuspy/siriuspy/devices/injsys.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py]: right, done!

File: siriuspy/siriuspy/devices/injsys.py (hunk 2 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py]: done!

File: siriuspy/siriuspy/devices/injsys.py [hunk 3]
```diff
@@ -591,16 +472,33 @@ def __init__(self, hltiming=None):
             self._on_values[dev] = {
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:472]: the loop here should be on `self._llrf_devs`, not `devices`

Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py:472]: right, done!

File: siriuspy/siriuspy/devices/injsys.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 from .lillrf import DevLILLRF
 from .modltr import LIModltr
 from .pwrsupply import PowerSupply, PowerSupplyPU
-from .timing import EVG, Event, Trigger
+from .timing import Trigger, HLTiming
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:12]: it seems the Trigger device is not used anymore

File: siriuspy/siriuspy/devices/injsys.py [hunk 5]
```diff
@@ -303,10 +189,11 @@ def __init__(self):
...
         self._psdevs = [PowerSupply(psn) for psn in self._psnames]
 
-        # trigger devices
-        self._trigdevs = [Trigger(trg) for trg in self._trignames]
+        # call base class constructor
+        super().__init__('', self._psdevs, hltiming=hltiming)
 
-        alldevs = tuple(self._psdevs + self._trigdevs)
+        # trigger devices
+        self._trigdevs = [self._hltiming.triggers(t) for t in self._trignames]
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:196]: bug: `[self._hltiming.triggers[t] for t in self._trignames]`

File: siriuspy/siriuspy/devices/injsys.py (hunk 4 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py]: right, I removed it. Thanks!

File: siriuspy/siriuspy/devices/injsys.py (hunk 5 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py]: done!

**Review by anacso17 — APPROVED** [review]

File: siriuspy/siriuspy/devices/injsys.py [hunk 6]
```diff
@@ -646,18 +680,17 @@ class InjSysStandbyHandler(_Devices):
         'as_pu': 'AS PU (Septa, Kickers and Modulators)',
         'bo_ps': 'BO PS Ramp',
         'bo_rf': 'BO RF Ramp',
-        'injbo': 'TI InjBO Event',
         'li_rf': 'LI LLRF (Klystrons Loop)',
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/injsys.py:683]: should be updated, maybe:
```
        'as_pu': 'PU Magnets (Septa and Kickers)',
        'bo_ps': 'BO PS Ramp',
        'bo_rf': 'BO RF Ramp',
        'li_rf': 'Linac devices (SHB, Klystrons and Modulators)',
```
?

Comment (fernandohds564) [reply siriuspy/siriuspy/devices/injsys.py]: right, done!

**Review by xresende — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Pull request merged**